packages/i18n/src/test/index.js

```diff
@@ -26,6 +26,10 @@ const localeData = {
 
 	"fruit\u0004%d apple" : [ "une pomme", "%d pommes" ],
 }
+const additionalLocaleData = {
+	"cheeseburger" : ["hamburger au fromage"]
```

Comment:
Code style:

 - Space between array square brackets ([reference](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#arrays-and-function-calls))
- No space before property colon ([reference](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#objects))
- No quotes on property key when not needed ([reference](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#objects))
- Single quote for strings ([reference](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/))